community/asset_library/using_assetlib.rst

```diff
@@ -21,19 +21,16 @@ AssetLib - at the moment, it's empty, as we are not logged in.
 Searching
 ~~~~~~~~~
 
-In the center is the **search bar + settings** section, and the **assets** section
-below it - this lets you filter out certain kinds of assets based on a host
-of criteria. These include the asset **category** (such as 2D tools, scripts
-and demos), **engine version** they are intended for, **sorting order** (by
-update date, by name, etc.) and **support level**.
+In the left sidebar is the **search bar + settings** section, and the **assets** section 
+takes up the main content area on the right. Below the search bar, you can filter assets 
+by **category** (such as 2D tools, scripts and demos), **engine version**, **license**, 
+**sorting order** (by update date, by name, etc.) and **support level**.
```

Comment:
```suggestion
**sorting order** (by update date, by name, etc.), and **support level**.
```
Please use [Oxford comma](https://docs.godotengine.org/en/latest/contributing/documentation/docs_writing_guidelines.html#use-the-oxford-comma-to-enumerate-anything).